frontend/src/scenes/onboarding/OnboardingBillingStep.tsx

```diff
@@ -57,7 +58,7 @@ export const OnboardingBillingStep = ({
                 )
             }
         >
-            {billing?.products && productKey && product ? (
+            {billing?.products && productKey && product && !billingLoading ? (
```

Comment:
Would it be easier to have billingloading be a top level check? 

`{billingLoading ? <Spinner> : .... }`